example/examplelibrary.py

```diff
@@ -34,4 +34,4 @@ def strings_should_be_equal(self, str1, str2):
 
 
 if __name__ == '__main__':
-    RobotRemoteServer(ExampleLibrary(), *sys.argv[1:])
+    RobotRemoteServer([ExampleLibrary()], *sys.argv[1:])
```

Comment:
Felt it is best to encourage/guide any new users of the remote server should use list in the example to avoid confusion going forward. Let me know if you think we should remove the list from this example.